dash/App/Notify/AddChain/index.js

```diff
@@ -240,49 +278,7 @@ class AddChain extends React.Component {
             </div>
 
             <div className='chainRow'>
-              {changedNewNetwork && newNetworkReady ? (
-                <div 
-                  className='addTokenSubmit addTokenSubmitEnabled' 
-                  onMouseDown={() => {
-                    const net = {
-                      id: this.state.newNetworkId,
-                      name: this.state.newNetworkName,
-                      type: this.state.newNetworkType,
-                      explorer: this.state.newNetworkExplorer,
-                      symbol: this.state.newNetworkSymbol,
-                      layer: this.state.newNetworkLayer,
-                      primaryRpc: this.state.newNetworkRPCPrimary,
-                      secondaryRpc: this.state.newNetworkRPCSecondary,
-                    }
-                    if (this.props.editMode) {
-                      const currentNet = {
-                        id: this.chain.id,
-                        name: this.chain.name,
-                        type: this.chain.type,
-                        explorer: this.chain.blockExplorerUrls[0],
-                        symbol: this.chain.nativeCurrency.symbol,
-                        layer: this.chain.layer,
-                        primaryRpc: this.state.newNetworkRPCPrimary,
-                        secondaryRpc: this.state.newNetworkRPCSecondary,
-                      }
-                      link.send('tray:action', 'updateNetwork', currentNet, net)
-                    } else {
-                      link.send('tray:addChain', net, this.props.req)
-                    }
-                    setTimeout(() => {
-                      link.send('tray:action', 'backDash')
-                    }, 400)
-                  }}
-                >
-                 {this.props.editMode ? 'Update Chain' : 'Add Chain'}
-                </div>
-              ) : (
-                <div 
-                  className='addTokenSubmit' 
-                >
-                  Fill in Chain
-                </div>
-              )}
+              {this.submitButton()}
```

Comment:
These should be dumb components, I'd pull submitButton (and its disabled variant) out into two SFCs and just pass the former the text it should use like 
```
<SubmitButton text={editMode ? 'Update Chain' : 'Add Chain'} />
```

Reply:
wasn't going to refactor stuff in this PR unless it was necessary. since we don't use this component anywhere else right now it should be fine. another thing I wanted to do was turn this into two components (`AddChain` and `EditChain`) but not sure it's worth the time and effort right now